angular-patternfly declares a dependency to `lodash-amd` and requires it using `require('lodash')`

This results in the error:

ERROR in ./~/angular-patternfly/index.js 
Module not found: Error: Cannot resolve module 'lodash' in /home/akieling/dev/projects/angularjs-styleguide/node_modules/angular-patternfly @ ./~/angular-patternfly/index.js 5:0-17

@alexkieling

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "angular-sanitize": "1.5.*",
     "angular-ui-bootstrap": "2.3.x",
     "angular-svg-base-fix": "2.0.0",
-    "lodash-amd": "4.x",
+    "lodash": "4.x",
     "patternfly": "3.25.0"
   },
   "devDependencies": {

